HarfangLab/harfanglab/ingest/parser.yml

```diff
@@ -187,3 +187,5 @@ stages:
           event.code: "{{json_event.message.event_id|string}}"
           action.properties: '{ {% for name, value in json_event.message.event_data.items() %}{% if value != None %}"{{name.replace(" ","")}}": "{{value | replace("\n", "") | replace("\\", "\\\\")}}",{% endif %}{% endfor %} }'
           action.id: "{{json_event.message.event_id}}"
+          user.target.name: "{{json_event.message.event_data.TargetUserName or json_event.message.event_data.SubjectUserName}}"
+          user.target.domain: "{{json_event.message.event_data.SubjectDomainName}}"
```

Comment:
IMHO, `SubjectUserName` should land in `user.name` and `SubjectDomainName` in `user.domain`. Only `TargetUserName` and `TargetUserDomain` should land in `user.target.*`.